section-09-differential-testing/model-package/tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+from regression_model.config.core import config
+from regression_model.processing.data_manager import load_dataset
+
+
+@pytest.fixture()
+def sample_input_data():
```

Comment:
don't we have a similar function in a previous file? do we need both?

Reply:
Good spot, removed

Reply:
The alternative would be to nest the diff test dir in the tests dir, but I'd like to keep them completely separate